test/test_lyrics.py

```diff
@@ -432,33 +431,90 @@ def setUp(self):
             self.skipTest("Python's built-in HTML parser is not good enough")
 
 
-class GeniusScrapeLyricsFromSongPageTest(GeniusBaseTest):
-    """tests Genius.scrape_lyrics_from_song_page()"""
+class GeniusScrapeLyricsFromHtmlTest(GeniusBaseTest):
+    """tests Genius._scrape_lyrics_from_html()"""
 
     def setUp(self):
         """Set up configuration"""
         GeniusBaseTest.setUp(self)
         self.plugin = lyrics.LyricsPlugin()
 
     def test_no_lyrics_div(self):
-        """Ensure that `lyrics_from_song_page` doesn't crash when the html
-        for a Genius page doesn't contain <div class="lyrics"></div>
+        """Ensure we don't crash when the scraping the html for a genius page
+        doesn't contain <div class="lyrics"></div>
         """
         # https://github.com/beetbox/beets/issues/3535
         # expected return value None
         url = 'https://genius.com/sample'
         mock = MockFetchUrl()
-        self.assertEqual(genius.scrape_lyrics_from_song_page(mock(url)), None)
+        self.assertEqual(genius._scrape_lyrics_from_html(mock(url)), None)
 
     def test_good_lyrics(self):
         """Ensure we are able to scrape a page with lyrics"""
         url = 'https://genius.com/Wu-tang-clan-cream-lyrics'
         mock = MockFetchUrl()
-        self.assertIsNotNone(genius.scrape_lyrics_from_song_page(mock(url)))
+        self.assertIsNotNone(genius._scrape_lyrics_from_html(mock(url)))
 
     # TODO: find an example of a lyrics page with multiple divs and test it
 
 
+class GeniusFetchTest(GeniusBaseTest):
```

Comment:
This is my first time really diving into unittests, so please let me know if there is a better way to do this